src/quanteda/generate_return_series.py

```diff
@@ -52,3 +52,34 @@ def generate_return_series(
     2024-01-02  0.043215  0.019105 -0.005555
     2024-01-03 -0.001011  0.003333 -0.011111
     """
+    np.random.seed(random_state)
+    if annual_volatility < 0:
+        raise ValueError("annual_volatility < 0.")
+
+    if freq == 'D':
+        periods_per_year = 365
+    elif freq == 'H':
+        periods_per_year = 365 * 24
+    elif freq == 'min':
+        periods_per_year = 365 * 24 * 60
+    else:
+        raise ValueError("Invalid frequency. Use 'D' for daily, 'H' for hourly, 'min' for minutely.") 
```

Comment:
This is something really small/nitpicky, but minutely isn't the right word to use here. Would the suggestion below work?
```suggestion
        raise ValueError("Invalid frequency. Use 'D' for day, 'H' for hour, 'min' for minute.") 
```